ors-api/src/main/resources/application.yml

```diff
@@ -165,3 +165,5 @@ springdoc:
   api-docs:
     path: /v2/api-docs
     version: OPENAPI_3_0
```

Comment:
Why not 3.1 (for documentation purposes)?

Reply:
Because with version 3.1 the `type: object` is not generated for parameters. So for now we should stay at Open API 3.0